src/MultivariateStats.jl

```diff
@@ -1,5 +1,6 @@
 module MultivariateStats
     using LinearAlgebra
+    using StatsBase: SimpleCovariance
```

Comment:
Put this in the StatsBase line below.

Reply:
That line below is an `import` statement, isn't it preferable to use `using` where it is sufficient?